azure-pipelines.yml

```diff
@@ -41,6 +41,24 @@ stages:
           - script: cargo test --all
             displayName: Cargo test
 
+          - script: |
+              cd docs && ../target/debug/zola build
+            condition: ne( variables['Agent.OS'], 'Windows_NT' )
+            displayName: Check documentation build
+          - script: |
+              cd docs && ..\target\debug\zola build
+            condition: eq( variables['Agent.OS'], 'Windows_NT' )
+            displayName: Check documentation build windows
+
+          - script: |
+              cd docs && ../target/debug/zola check
+            condition: ne( variables['Agent.OS'], 'Windows_NT' )
+            displayName: Check documentation links
+          - script: |
+              cd docs && ..\target\debug\zola check
+            condition: eq( variables['Agent.OS'], 'Windows_NT' )
+            displayName: Check documentation links windows
```

Comment:
That's no good, it would block the release of Zola if there is a dead link. I would be tempted to move `zola check` to GH actions actually.

Reply:
Hello @Keats, in fact that was the intention. :smile: 
To not pass the CI process if for any reason the documentation was broken or of boken links was introduced (Like the ones I corrected). Also it would check this on a regular basis and avoid documentation to deviate and keep it completely updated.
I like to put all checks in the CI, but if you think it annoying I can remove them from this PR.

Reply:
It's a good instinct but the link checker is not robust yet so that could cause me to comment out these lines in anger if I want to do a release and it's failing for whatever reason

Reply:
So I comment out the link check part.
I introduced nb_threads_per_cpu to configure the link_checker number of threads.